test/app-luatest/gh_6200_lua_cjson_escapes_slash_test.lua

```diff
@@ -0,0 +1,104 @@
+local compat = require('compat')
+local misc = require('test.luatest_helpers.misc')
+local jit = require('jit')
+local json = require('json')
+local popen = require('popen')
+local clock = require('clock')
+local t = require('luatest')
+local g = t.group()
+
+g.test_json_encode = function()
+    compat.json_escape_forward_slash = 'old'
+    -- Test that '/' is escaped with 'old' setting.
```

Comment:
Shouldn't we also check the default behavior here?

Reply:
Fixed.